### PR TITLE
Deduplicate instructions

### PR DESCRIPTION
**Pull request by mattias-p**
This is another step in my deduplication effort regarding installation-related materials.
Additional drive-by updates:
 * ~Grouped together miscellaneous topics under a common topic at the end of the document.~ (Moving to a future PR)
 * Gathered up some directory creation and file copying on FreeBSD.
 * Replaced some `echo | tee -a` commands with `sysrc` on FreeBSD.

### CODE REVIEW

File: docs/Installation.md [hunk 1]
```diff
@@ -536,10 +481,18 @@ service zm_rpcapi start
...
-### 3.4 Post-installation sanity check (FreeBSD)
 
-If you followed this instructions to the letter, you should be able to use the
-API on localhost port 5000, like this:
+## 4. Installation on Ubuntu
+
+Use the procedure for installation on [Debian](#2-installation-on-debian).
+
+
+## Post-installaiton
```
Comment (matsduf) [inline docs/Installation.md]: "installaiton" -> "installation"

**Comment by mattias-p** [comment]
There are lots of ways removing duplicated improves navigability, as well as maintainability and is generally a good thing. If you like I can enumerate a bunch of arguments.

The overview section is already a kind of a TOC. Replacing it with a bullet list would probably be an improvement.

I'm not a fan of the numbering. It messes with hyperlinks. Also we do have the OS suffixes in all the OS specific heading.

I'll just go ahead and push an update along these lines.

**Comment by matsduf** [comment]
@mattias-p, we need a document that is easy to navigate in.

I remember that numbering created problems with the links, but then I suggest removing the numbers from all headlines. I.e. make it consistent.

With your solution we need a table of contents at the start of the document pointing at the level one and level two headlines (level two and level three from a markdown perspective).

**Comment by mattias-p** [comment]
On second thought. Lets keep the discussion of the outline separate from this thread. I reduced the contents of this PR to only deal with file copying and sysrc.

**Pull request merged**